config/v1/types_cluster_version.go

```diff
@@ -691,6 +691,15 @@ type Update struct {
 // Release represents an OpenShift release image and associated metadata.
 // +k8s:deepcopy-gen=true
 type Release struct {
+	// architecture is an optional field that indicates the
+	// value of the cluster architecture. In this context cluster
+	// architecture means either a single architecture or a multi
+	// architecture.
+	// Valid values are 'Multi' and empty.
```

Comment:
This may very well end up extended with more detail in the near future, but starting here is acceptable.